index.html

```diff
@@ -32,9 +32,9 @@
     <script src="bower_components/urijs/src/URI.min.js"></script>
     <script src="js/zap-common.js"></script>
     <script src="js/everything.js"></script>
-    <script src="https://apis.google.com/js/api.js?onload=handleClientLoad"></script>
+    
     <script src="bower_components/plyr/dist/plyr.js"></script>
-
+    <script src="js/api.js"></script>
```

Comment:
Any clues on how to get rid of this error (which occurs when including this js file)
`1) should have required HTML elements` !?